### DISCUSSION TITLE
Miner run

### DISCUSSION THREAD

**Discussion by Sangmin1514** [filecoin-project/lotus#9581] [General Q&A] [answered]
https://lotus.filecoin.io/tutorials/lotus-miner/run-a-miner/

Setup - 5. Start the miner - lotus-miner run

After entering the command, the following error occurs.

[flexi_logger] [ERRORCODE: :Time] flexi_logger has to work with UTC rather than with local time, caused by IndeterminateOffset
    See https://docs.rs/flexi_logger/latest/flexi_logger/error_info/index.html#time

If anyone knows how to solve it, please. thank you

**Comment by rjan90** [comment, marked as answer]
Hey @Sangmin1514!

Yeah, this is a known issue that is coming from a dependency that the rust-fil-logger depends on. https://github.com/filecoin-project/lotus/issues/8618

The error seems to have been fixed in the underlying chrono-dependecy, we will need that fix to bubble up to `flexi_logger` which the rust-fil-logger depends on before we get a fix in Lotus. The error-message will not cause any issues though.

**Reply by Sangmin1514** [reply]
Oh! Thank you for your help as always. :)
